src/analyze.ts

```diff
@@ -19,6 +18,8 @@ import mapboxPregyp from '@mapbox/node-pre-gyp';
 import { Job } from './node-file-trace';
 import { fileURLToPath, pathToFileURL, URL } from 'url';
 
+const asyncWalk: typeof import('../node_modules/estree-walker').asyncWalk = require('estree-walker').asyncWalk
```

Comment:
Why is this using require instead of import?

Reply:
TypeScript was resolving to the `estree-walker` version used in `rollup-pluginutils` instead of the top-level `estree-walker` version which doesn't have `asyncWalk` so it's to fix the incorrect type

Reply:
Lets add a comment why this is here because we're going to forget 👍

Reply:
Added in https://github.com/vercel/nft/pull/236/commits/02db8aeb263ff2318dbe162c64c0fd45480df7e2